ldk-server-protos/src/proto/events.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+import "types.proto";
+package events;
+
+// Represents different types of events, useful for routing and filtering.
```

Comment:
Not sure what is meant by "useful for routing and filtering".

Reply:
Yeah, I guess I don't understand why switching / matching on the `oneof` can't be used. Seems like `event_type` is duplicating that.